### DISCUSSION TITLE
Add support for fixed columns.

### DISCUSSION THREAD

**Discussion by shivammathur** [Laravel-Backpack/community-forum#797] [Ideas]
In a large crud when responsive is set to false, it would improve UX if we can set the leftmost columns to fixed on horizontal scroll.
https://datatables.net/extensions/fixedcolumns/examples/initialisation/left_right_columns.html

**Comment by jcastroa87** [comment]
Hello @shivammathur 

It sounds like something useful to use, but at the moment it is not a feature that many developers request from us, in the same way we will wait if more developers require it.

Meanwhile you can use
```
php artisan backpack:publish crud/inc/datatables_logic
```

To publish datatables_logic.blade.php in your resources/views and modify as you need.

Cheers.

**Comment by jcastroa87** [comment]
Hi @shivammathur due to not activity i will close the issue, please feel free to reopen or create a new one.

Cheers.
